s3fs-fuse.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: s3fs-fuse
+  version: "1.94"
+  epoch: 0
+  description: FUSE-based file system backed by Amazon S3
+  copyright:
+    - license: GPL-2.0-only
+  # https://github.com/s3fs-fuse/s3fs-fuse/blob/master/COMPILATION.md#note---the-required-librariescomponents-required-to-run-s3fs-are
+  dependencies:
+    runtime:
+      - fuse2
+      - openssl
+      - curl
+      - libxml2
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl-dev
+      - fuse2-dev
+      - gcc
+      - gettext-dev
+      - git
+      - libtool
+      - libxml2-dev
+      - openssl-dev
```

Comment:
many of the above are redundant, i.e. git is pulled it automatically, and so is gcc, ca-certificates-bundle. It helps to have smaller list of things that are unique to this build. But that's a nitpick, doesn't have any material impact as all of these are installed anyway one way or another.